accord-core/src/main/java/accord/messages/ReadTxnData.java

```diff
@@ -218,6 +232,14 @@ void maybeRead(SafeCommandStore safeStore, SafeCommand safeCommand)
         }
     }
 
+    @Override
+    protected synchronized void readComplete(CommandStore commandStore, @Nullable Data result, @Nullable Ranges unavailable)
+    {
+        // TODO (expected): lots of undesirable costs associated with the obsoletion tracker
+//        commandStore.execute(contextFor(txnId), safeStore -> safeStore.command(txnId).removeListener(obsoleteTracker));
```

Comment:
can you remove this comment?

Reply:
Why? We probably want to address this if we can. I can reword it. I have for now commented out the logic that unregisters the obsoletion tracker, as it was incurring significant costs. Ideally we want to be able to manage these more cheaply.